python/cudf/cudf/benchmarks/pytest.ini

```diff
@@ -0,0 +1,14 @@
+[pytest]
```

Comment:
Can we use a `pyproject.toml` or `setup.cfg` instead of a `pytest.ini` file? The other two are standardized across tools.

Reply:
Using existing setup.cfg